book/installation.rst

```diff
@@ -244,12 +243,11 @@ If there are any issues, correct them now before moving on.
         $ sudo chmod +a "$HTTPDUSER allow delete,write,append,file_inherit,directory_inherit" app/cache app/logs
         $ sudo chmod +a "`whoami` allow delete,write,append,file_inherit,directory_inherit" app/cache app/logs
 
+    **3. Using ACL on a system that supports setfacl (most Linux/BSD)**
 
-    **3. Using ACL on a system that does not support chmod +a**
-
-    Some systems don't support ``chmod +a``, but do support another utility
+    Most Linux and BSD distributions don't support ``chmod +a``, but do support another utility
     called ``setfacl``. You may need to `enable ACL support`_ on your partition
-    and install setfacl before using it (as is the case with Ubuntu). This
+    and install setfacl before using it (as is the case with old versions of Ubuntu). This
```

Comment:
I think we should then simply remove this note. People using even older versions will very likely have other issues too.